framework/src/main/java/org/motechproject/uitest/page/DataServicesPage.java

```diff
@@ -43,14 +54,48 @@ public DataServicesPage createNewEntity(String entityName) throws InterruptedExc
     }
 
     /**
-     * Method that goes to data services page and enters entity table.
+     * Goes to schema editor page and enters entity
+     * @param entityName name of the entity we want to edit
+     */
+    public void goToEditEntity(String entityName) throws InterruptedException {
```

Comment:
I'm also thinking about splitting this page object into 3 - one for each tab. But maybe in a separate pr